tests/Bridge/Symfony/Routing/ApiLoaderTest.php

```diff
@@ -113,6 +134,24 @@ private function getApiLoaderWithResourceMetadata(ResourceMetadata $resourceMeta
 
         $kernelProphecy = $this->prophesize(KernelInterface::class);
         $kernelProphecy->locateResource(Argument::any())->willReturn($routingConfig);
+        $possibleArguments = [
+            'api_platform.action.get_collection',
+            'api_platform.action.post_collection',
+            'api_platform.action.get_item',
+            'api_platform.action.put_item',
+            'api_platform.action.delete_item',
+        ];
+
+        $containerInterfaceProphecy = $this->prophesize(ContainerInterface::class);
+        $containerInterfaceProphecy->reveal();
```

Comment:
or:

```php
$containerProphecy = $this->prophesize(ContainerInterface::class);
/** @var ContainerInterface $container */
$container = $containerProphecy->reveal();
// ...
$kernelProphecy->getContainer()->willReturn($container);
```

if you want to benefit from the typehint and avoid raising a warning on Scrutinizer.